pkg/osquery/table/keyinfo.go

```diff
@@ -0,0 +1,86 @@
+package table
```

Comment:
it seems a little weird to have 2 tables that are so similary, perhaps we could just allow for a path constraint on the kolide_ssh_keys table, and if a path is provided we could use that path instead of the os-specific defaults that we look in?

I might be missing an important distinction here, but the two tables seem to have a lot of overlap

Reply:
The intent is to be able to use the keyinfo table for any random keys. ssh, TLS, whatever. A fairly broad purpose "what is this key, and should I be worried about it being unencrypted".

The intent of the ssh keys table is to enumerate ssh keys more correctly than the upstream table does. 

I wouldn't want to combine the tables, but I absolutely want to have them using the same package